packages/ember-data/lib/system/model/model.js

```diff
@@ -295,7 +295,13 @@ var Model = Ember.Object.extend(Ember.Evented, {
     @property id
     @type {String}
   */
-  id: null,
+  get id () {
+    return this._internalModel.id;
+  },
+
+  set id (id) {
+    this._internalModel.setId(id);
+  },
```

Comment:
It looks the Ember object model doesn't like it when you use an enumerable computed getter. Instead of using the shorthand get and set syntax we will need to use `Object.defineProperty` at the bottom of this file to set the non enumerable computed id property.

```js
Object.defineProperty(Model.prototype, 'id', {
  configurable: true,
  enumerable: false,
  set(id) {
    if (this._internalModel) {
      this._internalModel.setId(id);
    }
  },
  get() {
    if (this._internalModel) {
      return this._internalModel.id;
    }
  }
});
```